appeals/web/src/server/appeals/appeal-details/application-outcome/__tests__/application-outcome.test.js

```diff
@@ -0,0 +1,70 @@
+import { parseHtml } from '@pins/platform';
+import supertest from 'supertest';
+import { appealData, appellantCaseDataNotValidated } from '#testing/app/fixtures/referencedata.js';
+import { createTestEnvironment } from '#testing/index.js';
+import nock from 'nock';
+
+const { app, installMockApi, teardown } = createTestEnvironment();
+const request = supertest(app);
+const appealId = appealData.appealId;
+const appellantCaseId = appealData.appellantCaseId;
+const baseUrl = `/appeals-service/appeal-details/${appealId}/appellant-case`;
+
+describe('application-outcome', () => {
+	beforeEach(installMockApi), afterEach(teardown);
+
+	describe('GET /change', () => {
+		it('shouldrender the applicationOutcome change page', async () => {
```

Comment:
Super pedantic but there's a missing space in the test name here: `shouldrender`